modules/Bio/EnsEMBL/Compara/RunnableDB/LoadMembers/CompareNonReusedGenomeList.pm

```diff
@@ -0,0 +1,120 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::LoadMembers::CompareNonReusedGenomeList
+
+=head1 SYNOPSIS
+
+    standaloneJob.pl Bio::EnsEMBL::Compara::RunnableDB::LoadMembers::CompareNonReusedGenomeList \
+        -compara_db $(mysql-ens-compara-prod-8 details url jalvarez_vertebrates_load_members_99) \
+        -expected_updates_file <(echo "homo_sapiens"; echo "mus_musculus"; echo "equus_caballus") \
+        -current_release 99 -nonreuse_ss_id 10000002
+
+=head1 DESCRIPTION
+
+This RunnableDB compares the list of genomes that are expected to have an
+annotation update (provided in a file) to the actual list of genomes that
+cannot be reused.
+
+Both lists must match, and the job fill fail if there are any references.
+If a difference happens to be fine, the species can be manually okay-ed
+by adding a parameter named "ok_${species_name}".
+Note: the Runnable respects the "do_not_reuse_list" parameter and won't
+complain that those species are not listed in the file.
```

Comment:
Since we were thinking about improving/updating our PODs, I was wondering if all the "non-eHive-runnable" arguments should be defined with `=item`. Would it be useful?

Reply:
Agreed. Will do that.